cmake/Corrosion.cmake

```diff
@@ -240,9 +239,9 @@ function(_add_cargo_build)
     set(byproducts)
     foreach(byproduct_file ${ACB_BYPRODUCTS})
         list(APPEND build_byproducts "${cargo_build_dir}/${byproduct_file}")
```

Comment:
What about this one?
There is also something similar line 154

Reply:
`cargo_build_dir` depends on both `build_dir` (which depends on $<CONFIG> for Multi-Config Generator) and on `build_type_dir`, which is either the cargo profile (if specified) or `debug` for debug builds or release for all other builds.

So the current behaviour is probably overly pessimistic and builds more seperately then it needs to.